src/olympia/lib/settings_base.py

```diff
@@ -1668,3 +1668,7 @@ def get_redis_settings(uri):
 SHELL_PLUS_POST_IMPORTS = (
     ('olympia', 'amo'),
 )
+
+DEFAULT_FXA_CONFIG_NAME = 'default'
+INTERNAL_FXA_CONFIG_NAME = 'internal'
```

Comment:
Yeah, for some reason I equated settings_base.py not importing settings.py with settings.py not importing settings_base.py. It's updated now.